storage/pebble/registers.go

```diff
@@ -79,17 +89,55 @@ func (s *Registers) Store(
 		return fmt.Errorf("failed to commit batch: %w", err)
 	}
 
+	// increment height if necessary
+	if height > s.latestHeight {
+		err = s.updateLatestHeight(height)
```

Comment:
any reason not to do this in the batch? this logic seems to assume that a whole block is indexed at once.

Reply:
code re-use mostly, but now that I can just remove the update functions below It'll be cleaner